packages/toolkit/package.json

```diff
@@ -4066,277 +4066,270 @@
                     "fontCharacter": "\\f1b4"
                 }
             },
-            "aws-amazonq-transform-landing-page-icon": {
```

Comment:
What are the changes on this file? Do you need to rebase maybe or something?

Reply:
The `package.json` inside toolkit directory is auto generated. For some reason this is get included. I'll remove this.